problems/2021-cnrs-benchmark/phase-1/buoyancy.i

```diff
@@ -302,33 +289,33 @@ t_alpha = 2e-4 # K-1, Thermal expansion coefficient
   [ntsApp]
     type = FullSolveMultiApp
     app_type = MoltresApp
-    execute_on = timestep_begin
+    execute_on = timestep_end
```

Comment:
Why this change?

Reply:
This makes the thermal-hydraulics problem run before the neutronics problem. I found that this order reduces simulation time. The final result is unchanged because this simulation terminates at steady state.